docs/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,160 @@
+openshift-samples                          4.3.0     True        False         False      42m
+operator-lifecycle-manager                 4.3.0     True        False         False      47m
+operator-lifecycle-manager-catalog         4.3.0     True        False         False      47m
+operator-lifecycle-manager-packageserver   4.3.0     True        False         False      45m
+service-ca                                 4.3.0     True        False         False      48m
+service-catalog-apiserver                  4.3.0     True        False         False      44m
+service-catalog-controller-manager         4.3.0     True        False         False      44m
+storage                                    4.3.0     True        False         False      43m
+support                                    4.3.0     True        False         False      48m
+```
+
+Operators which report `PROGRESSING=True` indicate that changes to the managed component are in the
+process of being rolled out. These changes can be new versions of component (in an upgrade), or
+changes applied to the component's desired configuration.
+
+Operators which report `DEGRADED=True` indicate that the managed component is operating at reduced
+level of service than what is desired. This does not mean that the component is out of service -
+however, degraded operators trigger an alert and generally require administrator intervention to
+resolve.
+
+Operators which report `AVAILABLE=False` indicate that the managed component is not operational.
+This means that one or more critical features of OKD will not work as desired. Operators which are
+not available will trigger an alert and generally require administrator intervention to resolve.
+
+If an operator is reporting `AVAILABLE=False` or `DEGRADED=True`, the cluster cannot be
+automatically upgraded.
+
+### Step 2: Identify Reason Operator Is Unhealthy
+
+```bash
+$ oc get clusteroperator image-registry -o yaml
+apiVersion: config.openshift.io/v1
+kind: ClusterOperator
+metadata:
+  creationTimestamp: "2019-12-05T15:45:19Z"
+  generation: 1
+  name: image-registry
+  resourceVersion: "28663"
+  selfLink: /apis/config.openshift.io/v1/clusteroperators/image-registry
+  uid: <uuid>
+spec: {}
+status:
+  conditions:
+  - lastTransitionTime: "2019-12-05T15:46:54Z"
+    message: The registry is ready
+    reason: Ready
+    status: "True"
+    type: Available
+  - lastTransitionTime: "2019-12-05T16:35:58Z"
+    message: The registry is ready
+    reason: Ready
+    status: "False"
+    type: Progressing
+  - lastTransitionTime: "2019-12-05T15:45:27Z"
+    status: "False"
+    type: Degraded
+  extension: null
+  relatedObjects:
+  - group: imageregistry.operator.openshift.io
+    name: cluster
+    resource: configs
+  - group: ""
+    name: openshift-image-registry
+    resource: namespaces
+  ...
+  versions:
+  - name: operator
+    version: 4.3.0
+```
```

Comment:
this would be more useful if it showed an example of a clusteroperator that was in an unhealthy state